api/src/twitterBot/bullQueue.ts

```diff
@@ -0,0 +1,56 @@
+import { FileData } from "./../utils";
+import { Worker, Queue, QueueScheduler } from "bullmq";
+import createDebug from "debug";
+import Redis from "ioredis";
+import { format, subDays } from "date-fns";
+import { tweet } from "./twit";
+import { microToStacks } from "../utils";
+
+const queueName = "tweet-stats";
+const debug = createDebug(`queue:${queueName}`);
+const redisClient = new Redis();
+
+export const tweetStatsQueueScheduler = new QueueScheduler(queueName, {
+  connection: redisClient,
+});
+export const tweetStatsQueue = new Queue<{}>(queueName, {
+  connection: redisClient,
+});
+
+const worker = new Worker(
+  queueName,
+  async (job) => {
+    const { currentData } = job.data;
```

Comment:
I removed the currentData when I pushed, it should be read directly from the file :)